services/movie_session.py

```diff
@@ -41,4 +41,9 @@ def update_movie_session(
 
 
 def delete_movie_session_by_id(session_id: int) -> None:
-    MovieSession.objects.get(id=session_id).delete()
+    MovieSession.objects.get_object_or_404(id=session_id).delete()
```

Comment:
`get_object_or_404()` is not a method on the `QuerySet` object. Use `MovieSession.objects.get(id=session_id)` and handle the exception if the session does not exist.